modules/app-gateway-module-backend/config.tf

```diff
@@ -41,7 +41,8 @@ data "azurerm_subnet" "app_gw" {
 }
 
 resource "azurerm_public_ip" "app_gw" {
-  name                = "aks-appgw-${var.env}-pip"
+  count               = length(var.private_ip_address)
```

Comment:
Can you remove changes to this module? They have been applied to the standalone module already.
In another PR we can look at removing this module from the folder

Reply:
These changes have been removed.